app/src/test/java/org/simple/clinic/home/overdue/OverdueScreenControllerTest.kt

```diff
@@ -128,32 +117,21 @@ class OverdueScreenControllerTest {
 
   @Test
   fun `when screen is created then the overdue appointments must be displayed`() {
+    // given
     val overdueAppointments = listOf(
         PatientMocker.overdueAppointment(riskLevel = OverdueAppointment.RiskLevel.HIGHEST),
-        PatientMocker.overdueAppointment(riskLevel = OverdueAppointment.RiskLevel.LOW),
+        PatientMocker.overdueAppointment(riskLevel = OverdueAppointment.RiskLevel.LOW, phoneNumber = null),
```

Comment:
Did you forget to remove `phoneNumber = null` from here?

Reply:
I originally left it in there to ensure appointments with no phone numbers are mapped correctly. Then I realised that the problem is that the property is nullable since we filter these out at the query level itself. I removed this for now; We'll need to make this property mandatory in the future,

Reply:
Yea I agree. This should not have been a `Nullable`.